src/Display.tsx

```diff
@@ -223,16 +223,19 @@ interface DisplayListProps {
 }
 
 const DisplayList: React.FC<DisplayListProps> = props => {
-  let content = props.dataArray.map((val: any, key: number) => (
-    <li key={key}>
-      <VictimDetail data={val.map}></VictimDetail>
-    </li>
-  ));
+  let content: any;
   if (props.dataArray.length === 0) {
-    content = <p>No data available.</p>;
-  }
-  if (!props.isReady) {
-    content = <p>Loading...</p>;
+    if (props.isReady === false) {
+      content = <Alert variant="danger" isInline title="No data available" />;
+    } else {
+      content = null;
+    }
```

Comment:
`props.isReady` is false when the API has not yet returned a response.
The `content` var might not be displaying the correct message in this condition.